spec/models/item_spec.rb

```diff
@@ -0,0 +1,5 @@
+require "rails_helper"
+
+RSpec.describe Item, type: :model do
```

Comment:
You don't have to specify the type. It's already inferred with config in `spec/rails_helper.rb`.